### ISSUE TITLE
Parse error on WiredCalendar "<<" with html-minifier

### ISSUE THREAD

**Issue by apennamen** [wiredjs/wired-elements#131]
Hello,

First thanks a lot for this library keep the good work!

#  TLDR
When building an application created with https://open-wc.org/ there is a Parse Error on the wired calendar. Wired calendar is using invalid HTML character "<" and ">" instead of "&lt;" and "&gt;"

# Environment
yarn v1.22.0
npm 6.13.4

# Steps to reproduce
First create an application using open wc utility
```
npm init @open-wc
√ What would you like to do today? » Scaffold a new project
√ What would you like to scaffold? » Application
√ What would you like to add? » Building (rollup or webpack)
√ What would you like to use for Building? » Rollup (recommended)
√ Would you like to scaffold examples files for? » (empty)
√ What is the tag name of your application/web component? ... wired-calendar-bug
√ Do you want to write this file structure to disk? » Yes
√ Do you want to install dependencies? » Yes, with yarn
```
Then add wired elements as dependency:
`yarn add wired-elements`

Modify `./src/WiredCalendarBug.js` and add import a Wired Element (doesn't have to be the calendar):
```javascript
// eslint-disable-next-line no-unused-vars
import { WiredButton } from 'wired-elements';
```

Finally build the project with `yarn build` results in the following error:
`[!] (plugin babel) Error: C:\Path\to\wired-calendar-bug\node_modules\wired-calendar\lib\wired-calendar.js: Parse Error: <<</th>`

# Suggested correction (not tested)
Replace "<<" with "&lt;&lt;"
https://github.com/wiredjs/wired-elements/blob/master/packages/wired-calendar/src/wired-calendar.ts#L197
Replace ">>" with "&gt;&gt;"

**Comment by pshihn** [comment]
PRs welcome :)
Haven't looked at much wired-calendar code it's all community contributed
